docs/quick_start/starting.rst

```diff
@@ -37,4 +37,16 @@ In both cases the user will be prompted to enter a password. It is possible, but
 
    The *start*, *status* and *help* commands do not require authentication.
 
+It is also possible to use certificate based authentication to login to the system. In this case the "fledge" command line utility should be passed the *-c* flag with the name of the certificate file to use to authenticate.
+
+.. code-block:: console
+
+   $ /usr/local/fledge/bin/fledge -c ~/.fledge/admin.cert stop
```

Comment:
Instead of `~/.fledge/admin.cert`, we can simply refer to the certificate path.

Reply:
> Instead of `~/.fledge/admin.cert`, we can simply refer to the certificate path.

We could, this is only an example and I wanted to avoid making specific reference to where Fledge stores certificates as it may well move in future.

Reply:
Here, I mean a placeholder for the certificate path.